### PR TITLE
Script improvements 1932727, 1927837, 1918151, #27

### PR DESCRIPTION
**Pull request by psykokwak4**

### CODE REVIEW

File: SetupCrossTenantRelationshipForTargetTenant.ps1 [hunk 1]
```diff
@@ -134,6 +164,11 @@ param
     [ValidateScript({ -not [string]::IsNullOrWhiteSpace($_) })]
     $TargetTenantDomain,
 
+    [Parameter(Mandatory = $true, HelpMessage='Target tenant id. This is azure ad directory id or external directory object id in exchange online.', ParameterSetName = 'TargetSetupAll')]
```
Comment (anshuldube) [inline SetupCrossTenantRelationshipForTargetTenant.ps1:167]: Update the help message.

File: SetupCrossTenantRelationshipForTargetTenant.ps1 [hunk 2]
```diff
@@ -107,8 +119,26 @@ param
...
+        })]
     [string]$KeyVaultAuditStorageAccountName,
 
+    [Parameter(Mandatory = $false, HelpMessage='Storage account location', ParameterSetName = 'TargetSetupAll')]
+    [Parameter(Mandatory = $false, HelpMessage='Storage account location', ParameterSetName = 'TargetSetupAzure')]
+    [string]$KeyVaultAuditStorageAccountLocation,
+
+    [Parameter(Mandatory = $false, HelpMessage='Storage account SKU', ParameterSetName = 'TargetSetupAll')]
+    [Parameter(Mandatory = $false, HelpMessage='Storage account SKU', ParameterSetName = 'TargetSetupAzure')]
+    [string]$KeyVaultAuditStorageAccountSKU,
```
Comment (anshuldube) [inline SetupCrossTenantRelationshipForTargetTenant.ps1:140]: Can you add an example for this too?

File: SetupCrossTenantRelationshipForTargetTenant.ps1 [hunk 3]
```diff
@@ -282,6 +320,8 @@ function Create-KeyVaultAndGenerateCertificate([string]$targetTenant, `
                                                $retrieveCertPrivateKey, `
                                                [string]$auditStorageAcntRG, `
                                                [string]$auditStorageAcntName, `
+                                               [string]$auditStorageAcntLocation, `
```
Comment (anshuldube) [inline SetupCrossTenantRelationshipForTargetTenant.ps1:323]: can we use the same location as kv location?

Comment (psykokwak4) [reply SetupCrossTenantRelationshipForTargetTenant.ps1:323]: I think they are different, I need to use `westus` as storage account's location, and `West US` as kv location..

**Comment by psykokwak4** [comment]
#27

**Comment by psykokwak4** [comment]
sorry.. accidentally pushed to main branch, added PR Feedback to main..

**Pull request merged**